curriculum/challenges/english/01-responsive-web-design/css-picasso-painting/part-035.md

```diff
@@ -0,0 +1,154 @@
+---
+id: 60b69a66b6ddb80858c51599
+title: Part 35
+challengeType: 0
+dashedName: part-35
+---
+
+# --description--
+
+Within your new `black-character` element, add three `div` elements with the following `id` values, in order: `black-hat`, `gray-mask`, `white-paper`.
+# --hints--
```

Comment:
Just because I am a stickler.

```suggestion
Within your new `black-character` element, add three `div` elements with the following `id` values, in order: `black-hat`, `gray-mask`, `white-paper`.

# --hints--
```